src/main/java/ca/unknown/bot/use_cases/StudyInteractor.java

```diff
@@ -0,0 +1,193 @@
+package ca.unknown.bot.use_cases;
+
+import ca.unknown.bot.data_access.JSONQuizMeRepository;
+import ca.unknown.bot.entities.QuizMe;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.component.ButtonInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.ModalInteractionEvent;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.api.interactions.components.ActionRow;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.interactions.components.text.TextInput;
+import net.dv8tion.jda.api.interactions.components.text.TextInputStyle;
+import net.dv8tion.jda.api.interactions.modals.Modal;
+
+import java.io.File;
+import java.util.Objects;
+
+public class StudyInteractor extends ListenerAdapter {
+
+
+
+    private QuizMe quizMe = new QuizMe();
+    private JSONQuizMeRepository repository = new JSONQuizMeRepository();
+
+    /**
+     * Handles all study commands
+     *
+     * @param event represents a SlashCommandInteraction event.
+     */
+    @Override
+    public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
```

Comment:
i think everything past the onSlashCommandInteraction method should be in dif classes to follow CA more closely